js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -1590,8 +1592,16 @@ function processQuery(query, explain, planIndex) {
         if (node.options && node.options.hasOwnProperty('parallelism') && node.options.parallelism > 1) {
           rc += annotation(' /* parallelism: ' + node.options.parallelism + ' */');
         }
+        if (node.options && node.options.order) {
+          let order = node.options.order;
+          if (node.options.order === 'weighted') {
+            order += ', weight attribute: ' + node.options.weightAttribute;
+          }
+          rc += annotation(' /* order: ' + order + ' /*');
```

Comment:
```suggestion
          rc += annotation(' /* order: ' + order + ' */');
```

Not sure but maybe typo?